ext/search_kit/Civi/Api4/Action/SearchDisplay/AbstractRunAction.php

```diff
@@ -303,9 +303,10 @@ private function formatColumn($column, $data) {
    * @return string
    */
   private function rewrite(array $column, array $data): string {
-    // Cheap strpos to skip Smarty processing if not needed
-    $hasSmarty = strpos($column['rewrite'], '{') !== FALSE;
     $output = $this->replaceTokens($column['rewrite'], $data, 'view');
+    $output = \Civi\Token\TokenCompatSubscriber::renderConditionalPunctuation($output);
```

Comment:
@aydun it looks like this will process conditional puctuation for the square-bracket tokens but not the smarty tokens?
If you moved this line to the bottom might it handle both?

Reply:
The problem is that the conditional punctuation looks like invalid smarty and causes errors.  There might be a way to change that behaviour and pass them through as is.  Or alternatively use something other than `{}` and avoid it looking like smarty, but then it's not consistent with other uses.